etc/p3-config/p3-config.yml

```diff
@@ -707,3 +718,35 @@ p3_container_clusters_template_k8s_fedora_25:
   # docker_volume_size must be > 3 to trigger creation of a volume.
   docker_volume_size: 3
   server_type: "bm"
+
+# Docker swarm magnum cluster template.
+p3_container_clusters_template_swarm_fedora_atomic_27:
+  external_network: "{{ p3_network_ilab_name }}"
+  fixed_network: "{{ p3_network_internal_name }}"
+  fixed_subnet: "{{ p3_network_internal_name }}"
+  master_flavor: "{{ p3_flavor_compute_A.name }}"
+  flavor: "{{ p3_flavor_compute_A.name }}"
+  image: "FedoraAtomic27"
+  name: "swarm-fedora-atomic-27"
+  coe: "swarm-mode"
+  network_driver: "docker"
+  docker_storage_driver: "overlay2"
+  # docker_volume_size must be > 3 to trigger creation of a volume.
+  docker_volume_size: 3
+  server_type: "vm"
```

Comment:
This one probably deserves a comment to the effect that we're using the standard VM driver which now works for bare metal.